src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
```

Comment:
```suggestion
export const CommentInfo = ({ comment: name, email, body }) => (
```
Consider using restructuring here